en/docs/references/statement-editor/statement-editor-overview.md

```diff
@@ -115,6 +115,6 @@ Follow the steps below to build the statement.
 
 To explore more on the Statement Editor functionalities, see the use cases below.
 
-- [Call a Ballerina library function](../references/statement-editor/call-a-ballerina-library-function.md)
-- [Construct a record](../references/statement-editor/construct-a-record.md)
-- [Define a configurable variable](../references/statement-editor/define-a-configurable-variable.md)
+- [Call a Ballerina library function](../../references/statement-editor/call-a-ballerina-library-function.md)
```

Comment:
we can make this `call-a-ballerina-library-function.md` as well isn't it. Shorter.
Will send a separate PR for that